howso/utilities/feature_attributes/relational.py

```diff
@@ -804,6 +804,18 @@ def _infer_feature_bounds(self,  # noqa: C901
             column_type, _ = self._parse_column_type(str(column.type))
             format_dt = None
 
+            # Compute loose time-only feature bounds
+            if feature_attributes[feature_name]['data_type'] == 'formatted_time':
+                if (
```

Comment:
Should there also be an else branch here like in the pandas variant to handle tight bounds min/max?